src/main/java/com/microsoft/cse/helium/app/dao/ActorsDao.java

```diff
@@ -14,54 +13,29 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
 
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
-import javax.xml.ws.soap.AddressingFeature;
-import java.util.regex.Pattern;
-
-
 @Service
 public class ActorsDao extends BaseCosmosDbDao {
 
     private static final Logger _logger = LoggerFactory.getLogger(ActorsDao.class);
-
     final String _actorSelect = "select m.id, m.partitionKey, m.actorId, m.type, m.name, m.birthYear, m.deathYear, m.profession, m.textSearch, m.movies from m where m.type = 'Actor' ";
     final String _actorContains = " and contains(m.textSearch, \"%s\") ";
     final String _actorOrderBy = " order by m.name ";
     final String _actorOffset = " offset %d limit %d ";
 
     final String _actorSelectById = _actorSelect + " and m.actorId = '%s'";
 
-
-    @Autowired
```

Comment:
Does the configService still get injected with Autowired removed?

Reply:
Yes